kod/object/passive/trestype/mimict.kod

```diff
@@ -0,0 +1,43 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+MimicTreasure is TreasureType
+
+constants:
+
+   include blakston.khd
+
+classvars:
+
+
+properties:
+
+   piTreasure_num = TID_MIMIC
```

Comment:
Recent change in dev branch means piTreasure_num should be in classvars now, and called viTreasure_num